### ISSUE TITLE
Unable to reach top of conversation

### ISSUE THREAD

**Issue by ekkis** [wireapp/wire-desktop#4046]
Wire version: 3.16.3630
Operating system: OSX 10.14.6

What steps will reproduce the problem?

1.  scroll to top of window
2.  wait for previous messages to load
3.  scroll again

What is the expected result?

more messages

What is the actual result?

the software slows down to the point of becoming unresponsive

---

I have a really long conversation.  I want to export/back it up, but Wire apparently offers no means of achieving that so I'm going to just copy/paste the content into an external file.  the problem is I can't reach the top of the conversation as Wire slows down the more I scroll to the point it simply stops responding

Wire also does not provide an icon or other means to navigate to the top of the conversation so scrolling is all I have

can someone suggest how I can export/save my conversation, which is what I actually want (and no, please do not suggest the backup functionality, which produced unreadable, unusable backup files)?

**Comment by ffflorian** [comment]
Hi @ekkis,

> I want to export/back it up, but Wire apparently offers no means of achieving that

Did you try the backup? https://support.wire.com/hc/en-us/articles/360000824805-Back-up-your-conversation-history

**Comment by ekkis** [comment]
yes.  the backup may work but the restore doesn't.  but in any case, that's not the topic of this issue
